docker-compose.yml

```diff
@@ -0,0 +1,63 @@
+version: "3"
+services:
+  traefik:
+    container_name: traefik
+    image: traefik:v2.9.6
+    command:
+      - "--providers.docker=true"
+      - "--providers.docker.exposedbydefault=false"
+      - "--entrypoints.web.address=:80"
+    ports:
+      - "80:80"
+    networks:
+      - web
+    volumes:
+      - /var/run/docker.sock:/var/run/docker.sock
+
+  nomis-portal:
+    build:
+      context: .
+      dockerfile: Dockerfile
+    labels:
+      - "traefik.enable=true"
+      - "traefik.http.routers.nomis_portal.rule=Host(`nomis.local`)"
+      - "traefik.http.services.nomis_portal.loadbalancer.server.port=3838"
+    environment:
+      NOMIS_ENV: prod
+      NOMIS_DB_HOSTNAME: nomis-db
+      NOMIS_DB_PORT: 3306
+      NOMIS_DB_USERNAME: nomis_portal
+      NOMIS_DB_PASSWORD: nomis
+      NOMIS_DB_NAME: nomis_db
```

Comment:
I would inject such content from environment variables.
Usualy, we have a `.env` file for this.